### DISCUSSION TITLE
Time/Period specific search results?

### DISCUSSION THREAD

**Discussion by rohitrsm83** [searxng/searxng#3769] [Q&A] [answered]
Greetings,

A lay user here and very happy with SearX.

Is there a way to do a time/period specific search (like we can do in DDG e.g. display results from 1.1.2005 to 1.1.2008). If this feature is there, I would love to use it. 

Please advise.

**Comment by Austin-Olacsi** [comment, marked as answer]
nope. there is a feature request, see https://github.com/searxng/searxng/issues/682

however, if a given search engine has search operators for date range that would work. for example:

!google apple after:YYYY-MM-DD before:YYYY-MM-DD

the above will search "apple" on google within the date range. It appears duckduckgo can't be used this way.

**Reply by rohitrsm83** [reply]
Thank you, this is very helpful.

**Mod/Admin may close this thread.**
